bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -127,14 +125,18 @@ public synchronized void login() {
             return;
         }
 
+        if (tooManyRequests || ThingStatus.ONLINE.equals(thing.getStatus())) {
```

Comment:
Because `ThingStatus` is an enum so you can use `==` in comparisons:

```suggestion
        if (tooManyRequests || ThingStatus.ONLINE == thing.getStatus()) {
```

Reply:
fixed